saleor/discount/utils/promotion.py

```diff
@@ -371,6 +262,40 @@ def _update_promotion_discount(
     )
 
 
+def _update_catalogue_promotion_discount(
```

Comment:
Looks like the function is missing tests 🚔

Reply:
This function is not used at all. Let me drop it